src/service/auth.ts

```diff
@@ -49,8 +53,9 @@ const validate = (authToken: string | undefined, requiredScopes?: string[]): str
 export class AuthenticationService {
     authToken?: string;
     zeplinApi = new ZeplinApi();
+    loginServer = new LoginAuthServer(defaults.app.authRedirectPath);
 
-    async authenticate(params?: { requiredScopes: string[] }): Promise<string> {
+    async authenticate(params?: { requiredScopes: string[]; noBrowser: boolean }): Promise<string> {
```

Comment:
```suggestion
    async authenticate({requiredScopes, noBrowser = false}?: { requiredScopes: string[]; noBrowser: boolean } = {}): Promise<string> {
```

To prevent null checks and setting default values in the function body